upgrades/schema/Version_4_0_20210331092434_add_lock_table.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version_4_0_20210331092434_add_lock_table extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $sql = <<<SQL
+CREATE TABLE lock_keys (
```

Comment:
To create on the fly:
```suggestion
CREATE TABLE IF NOT EXISTS lock_keys (
```